docker/experimental/Dockerfile

```diff
@@ -0,0 +1,122 @@
+ARG python_version=3.8
+ARG debian_version=buster
+
+ARG cuda=0
+ARG cuda_version=11.2
+ARG cudnn_version=8.1.1.33
+# Debian repo doesn't have libcudnn
+ARG cuda_repo=ubuntu1804
+ARG tf_cuda_compute_capabilities="7.0"
```

Comment:
given https://github.com/pytorch/xla/issues/4089, should we support more `tf_cuda_compute_capabilities` ?

Reply:
Good idea. I'll merge that change.